lib/utils.js

```diff
@@ -195,7 +195,7 @@ exports.mixin = (target, b) => {
     if (desc.hasOwnProperty('value')) {
       if (target.hasOwnProperty(key) && isObject(desc.value)) {
         let existing = Object.getOwnPropertyDescriptor(target, key);
-        if (isObject(existing.value)) {
+        if (isObject(existing.value) && existing.value !== desc.value) {
```

Comment:
what if `value` is Boolean `false`?

Reply:
then it'll fail the `isObject()` tests and never hit this line..?

i'm a bit fuzzy on what `mixin` needs to do as compared to something like `Object.assign()` tbh - happy to toss some tests at it if you have specific use-cases in mind.

Reply:
Sorry for the late reply. I remember making this comment on my phone, I must have been looking at something else before this. The code I thought I was commenting on did `if (!value)`. Sorry for the confusion. 

I think your code makes sense. I'll look into the CI failure.